src/platform_impl/windows/event_loop.rs

```diff
@@ -1734,17 +1697,12 @@ unsafe fn public_window_callback_inner<T: 'static>(
                 Some(cursor) => {
                     let cursor = winuser::LoadCursorW(ptr::null_mut(), cursor.to_windows_cursor());
                     winuser::SetCursor(cursor);
-                    0
+                    result = ProcResult::Value(0);
                 }
-                None => winuser::DefWindowProcW(window, msg, wparam, lparam),
+                None => result = ProcResult::DefWindowProc,
             }
         }
 
-        winuser::WM_DROPFILES => {
-            // See `FileDropHandler` for implementation.
-            0
-        }
-
```

Comment:
Why is this being removed again? It's present in #1788 as well.

Reply:
Because it's handled by this:
https://github.com/rust-windowing/winit/pull/2143/files#diff-edacb93792c7cf2d4cfe6f4ff2259781ed3c13e060fcc381e9cbb30ce481a336R780